docs/plugins/subtitles.md

```diff
@@ -0,0 +1,144 @@
+# Subtitles
+
+You can use the *Subtitles* plugin to easily display Subtitles (or Closed Captions) in your App.
+
+The plugin comes with some basic styling of Subtitles and offers the option to customize them, such as font size, colors, and positioning.
+
+> The Subtitles plugins in the Lightning-SDK only offers an API for displaying Subtitles.
+
+> Retrieving subtitles from a video stream or a remote API is not part of the Plugin's functionality as it's very App and / or Platform specific.
+
+## Usage
+
+If you want to display Subtitles in your App, first import the Subtitles plugin from the Lightning SDK:
+
+```js
+import { Subtitles } from '@lightningjs/sdk'
+```
+
+## Available Methods
+
+### show
+
+Sets the visibility of Subtitles to `true`, to display Subtitles.
+
+```js
+Subtitles.show()
+```
+
+### hide
+
+Sets the visibility of Subtitles to `false`, to hide Subtitles.
+
+```js
+Subtitles.hide()
+```
+
+### text
+
+Sets the text of the Subtitles text.
+
+```js
+Subtitles.text('Subtitle Text')
+```
+
+### clear
+
+Clears the text of the Subtitles text.
+
+```js
+Subtitles.clear()
+```
+
+### styles
+
+Sets the styles of the Subtitles text. This is a short hand method for setting the following styles: `fontFamily`, `fontSize`, `fontColor`, `backgroundColor`, `textAlign`. If any of these styles are not set, the default values will be used.
+
+```js
+Subtitles.styles({
+  fontFamily: 'sans-serif',
+  fontSize: 45,
+  fontColor: 0xffffffff,
+  backgroundColor: 0x90000000,
+  textAlign: 'center',
+})
+```
+### fontFamily
+
+Sets the `fontFamily` style of the Subtitles text.
+
+```js
+Subtitles.fontFamily('sans-serif')
+```
+
+### fontSize
+
+Sets the `fontSize` style of the Subtitles text.
+
+```js
+Subtitles.fontSize(50)
+```
+
+### fontColor
+
+Sets the `fontColor` style of the Subtitles text
+
+```js
+Subtitles.fontColor(0xffffffff)
+```
```

Comment:
If you opt to change this property to textColor, just a reminder to also change the DOCS.

Reply:
The same explanation regarding `fontColor` is also valid here : https://github.com/rdkcentral/Lightning-SDK/pull/373#discussion_r1107165496